packages/grafana-ui/src/components/Forms/InlineFieldRow.mdx

```diff
@@ -0,0 +1,14 @@
+Used to align multiple inline form fields in one row. The row will wrap if the width of the children exceeds its own. Equivalent to the div with `gf-form-inline` class name.
```

Comment:
> The row will wrap if the width of the children exceeds its own.
I kind of know what you mean but this is still a bit confusing to me. Maybe change to "If the width of the children exceeds the row width, the row will wrap." Also I guess a screenshot example of how wrapped rows look would be nice. 

You explain that a row stacks multiple fields, but you don't address using multiple rows and it's not in the code example either. I guess it's fine to just put multiple rows one after another? Also there is no props table for this component, is that on purpose?

Just like in the InlineField docs, I would list related components and how they can and should be used within this component.

Reply:
Yeah, there's no props for this component except the native HTML ones and storybook doesn't display those for some reason.